workos/table_workos_directory.go

```diff
@@ -0,0 +1,184 @@
+package workos
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"github.com/workos/workos-go/v2/pkg/directorysync"
+)
+
+func tableWorkOSDirectory(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "workos_directory",
+		Description: "Retrieve information about your directories.",
+		List: &plugin.ListConfig{
+			Hydrate: listDirectories,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "domain",
+					Require: plugin.Optional,
+				},
+				{
+					Name:    "name",
+					Require: plugin.Optional,
+				},
+				{
+					Name:    "organization_id",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getDirectory,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Type:        proto.ColumnType_STRING,
+				Description: "Directory unique identifier.",
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Description: "Directory name.",
+			},
+			{
+				Name:        "state",
+				Type:        proto.ColumnType_STRING,
+				Description: "Linked status for the Directory.",
+			},
+			{
+				Name:        "organization_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "Identifier for the Directory's Organization.",
```

Comment:
```suggestion
				Description: "Identifier of the directory's organization.",
```
@bigdatasourav could we please update all the table descriptions to follow the above suggestion?

Reply:
@misraved The descriptions are based on the API doc provided by WorkOS.

- Should we change them? I rechecked them, but there were no grammatical errors.
- If yes, please share which pattern or standard we should follow to write the column descriptions in upcoming plugins.